dojo/tools/generic/parser.py

```diff
@@ -30,12 +30,27 @@ def get_findings_json(self, filename, test, active=None, verified=None):
         data = json.load(filename)
         findings = list()
         for item in data['findings']:
+            # remove endpoints of the dictionnary
+            unsaved_endpoints = None
+            if "endpoints" in item:
+                unsaved_endpoints = item["endpoints"]
+                del item["endpoints"]
+
```

Comment:
also add clean() as per kibliks PR?

Reply:
I think it's better to add clean in the (re)importer
People that implement parsers will forget it.
Also do it in (re)importer will make things at the same level as the unit tests. I mean it's the object that use parsers and results of `get_findings()` that checks endpoints.

Reply:
sorry I meant in the testcase, but it's already there I see.